Polyhedron/demo/Polyhedron/Scene_polygon_soup_item.cpp

```diff
@@ -183,6 +221,7 @@ Scene_polygon_soup_item::triangulate_polygon(Polygons_iterator pit) const
     P_traits cdt_traits(normal);
 
     CDT cdt(cdt_traits);
+    QMap<CDT::Vertex_handle, int> p2p;
```

Comment:
Here again the name `p2p` is surprising. Imaging you want to re-read your own code next year. What variable name, or comment, should you use/add to help understanding the code better?